WatchApp Extension/Extensions/CLKComplicationTemplate.swift

```diff
@@ -62,7 +62,7 @@ extension CLKComplicationTemplate {
         let isGlucoseStale = date.timeIntervalSince(glucoseDate) > recencyInterval
 
         if isGlucoseStale {
-            glucoseString = "---"
+            glucoseString = NSLocalizedString("---", comment: "No glucose value representation (3 dashes for mg/dL; no spaces as this will get truncated in the watch complication)")
```

Comment:
Does this need to change, too?

Reply:
Unfortunately, in the watch complication, there isn't enough space for the update and no spaces doesn't look right.